src/app.js

```diff
@@ -1 +1,58 @@
 'use strict';
+/* eslint-disable no-console */
+/* eslint-disable max-len */
+
+const fs = require('fs');
+const path = require('path');
+
+const readline = require('readline').createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+readline.question('Enter the path of the file to copy: ', sourceFilePath => {
+  readline.question('Enter the destination directory: ', destinationFolderPath => {
+    readline.close();
+
+    copyFile(sourceFilePath, destinationFolderPath);
+  });
+});
+
+function copyFile(sourceFilePath, destinationFolderPath) {
+  const sourceFileName = path.basename(sourceFilePath);
+  const destinationPath = path.join(destinationFolderPath, sourceFileName);
+
+  if (!fs.existsSync(sourceFilePath)) {
+    console.error('Source file does not exist.');
+
+    return;
+  }
+
+  if (!fs.existsSync(destinationFolderPath)) {
+    console.error('Destination folder does not exist.');
+
+    return;
+  }
+
+  const destinationStats = fs.statSync(destinationFolderPath);
```

Comment:
You are using fs.statSync() which is a synchronous operation and can block the execution of the code. It is better to use fs.stat() which is an asynchronous operation.